src/integration-test/tests/legal/amountClaim/steps/amountClaims.ts

```diff
@@ -1,21 +1,26 @@
-'use strict'
-
-let personalInjuryPage, housingDisrepairPage, summariseTheClaimPage, amountPage, totalPage, detailsSummaryPage, statementOfTruthPage, payByAccountPage, submittedPage
-
-module.exports = {
-  _init () {
-    personalInjuryPage = require('../../amountClaim/pages/personal-injury')
-    housingDisrepairPage = require('../../amountClaim/pages/housing-disrepair')
-    summariseTheClaimPage = require('../../amountClaim/pages/summarise-the-claim')
-    amountPage = require('../../amountClaim/pages/amount')
-    totalPage = require('../../amountClaim/pages/total')
-    detailsSummaryPage = require('../../amountClaim/pages/details-summary')
-    statementOfTruthPage = require('../../amountClaim/pages/statement-of-truth')
-    payByAccountPage = require('../../amountClaim/pages/pay-by-account')
-    submittedPage = require('../../amountClaim/pages/submitted')
-  },
-
-  openPage  () {
+import { ClaimPersonalInjuryPage } from 'integration-test/tests/legal/amountClaim/pages/personal-injury'
+import { ClaimHousingDisrepairPage } from 'integration-test/tests/legal/amountClaim/pages/housing-disrepair'
+import { ClaimSummariseTheClaimPage } from 'integration-test/tests/legal/amountClaim/pages/summarise-the-claim'
+import { ClaimAmountPage } from 'integration-test/tests/legal/amountClaim/pages/amount'
+import { ClaimTotalPage } from 'integration-test/tests/legal/amountClaim/pages/total'
+import { ClaimDetailsSummaryPage } from 'integration-test/tests/legal/amountClaim/pages/details-summary'
+import { ClaimStatementOfTruthPage } from 'integration-test/tests/legal/amountClaim/pages/statement-of-truth'
+import { ClaimPayByAccountPage } from 'integration-test/tests/legal/amountClaim/pages/pay-by-account'
+import { ClaimSubmittedPage } from 'integration-test/tests/legal/amountClaim/pages/submitted'
+
+const personalInjuryPage: ClaimPersonalInjuryPage = new ClaimPersonalInjuryPage()
+const housingDisrepairPage: ClaimHousingDisrepairPage = new ClaimHousingDisrepairPage()
+const summariseTheClaimPage: ClaimSummariseTheClaimPage = new ClaimSummariseTheClaimPage()
+const amountPage: ClaimAmountPage = new ClaimAmountPage()
+const totalPage: ClaimTotalPage = new ClaimTotalPage()
+const detailsSummaryPage: ClaimDetailsSummaryPage = new ClaimDetailsSummaryPage()
+const statementOfTruthPage: ClaimStatementOfTruthPage = new ClaimStatementOfTruthPage()
+const payByAccountPage: ClaimPayByAccountPage = new ClaimPayByAccountPage()
+const submittedPage: ClaimSubmittedPage = new ClaimSubmittedPage()
+
+export class AmountClaimSteps {
+  // todo never used? remove
+  openPage () {
```

Comment:
Agree to remove it. That method does not make sense as it opens 9 different pages one by one.

Reply:
done